app/minutes/controllers.py

```diff
@@ -44,7 +44,7 @@ def get_minute(user, user_data):
 
     membership = committee.members.filter_by(member = user).first()
 
-    if minute.private and (membership is None and not user.is_admin and committee.head != user.id):
+    if minute.private and (membership is None and not user.is_admin):
```

Comment:
Logic was changed to be that any committee members or admins can view private minutes, so we don't need to check if a member is the committee head.